console/tasks/fix_linux_permissions.sh

```diff
@@ -22,7 +22,7 @@ match_user_id_between_host_and_container ()
 {
     SERVICE=$1
     CONTAINER_UID=$(${DOCKER_ROOT_COMMAND} ${SERVICE} sh -c "getent passwd ${USER_PHP} | cut -f3 -d:")
-    HOST_UID=$(${DOCKER_ROOT_COMMAND} ${SERVICE} sh -c "stat -c '%u' '.'")
```

Comment:
I guess this command was like that for a reason but I do not remember anymore. I trust that you tested it properly on a linux machine, so I will merge it ;)